tasks/src/main/java/taf/product/yandex/disk/util/StringUtils.java

```diff
@@ -0,0 +1,13 @@
+package taf.product.yandex.disk.util;
+
+public class StringUtils {
```

Comment:
One more StringUtils with only constants inside?

Reply:
I'm using DataStorage files for three different products